docs/source/examples/tex.rst

```diff
@@ -1,6 +1,27 @@
 LaTeX
 =================================
 
+The Tex() mobject
++++++++++++++++++++
+Just as you can use :class:`~.Text` to add text to your videos, you can use :class:`~.Tex` to insert LaTeX.
+
+.. manim:: ExampleLaTeX
+    :save_last_frame:
+
+    class ExampleLaTeX(Scene):
+        def construct(self):
+            tex = Tex(r'\LaTeX').scale(3)
+            self.add(tex)
+
+Note that we are using a raw string (r'---') instead of a regular string ('---').
+This is because TeX code uses a lot of special characters - like `\\` for example - 
```

Comment:
```suggestion
Note that we are using a raw string (``r'---'``) instead of a regular string (``'---'``).
This is because TeX code uses a lot of special characters - like ``\`` for example - 
```